More module documentation (#815)
kyma-project/btp-manager: docs/user/tutorials/04-30-install-module.md

# Install the SAP BTP Operator Module

<!--this content is for OS users only-->
To install the SAP BTP Operator module from the latest release, you must install BTP Manager and the SAP BTP service operator.

### Prerequisites

* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
* Kubernetes cluster, or [k3d](https://k3d.io) for local installation
* [jq](https://github.com/stedolan/jq)
* `sap-btp-manager` Secret created. See [Create `sap-btp-manager` Secret](04-10-create-btp-manager-secret.md).
  > [!NOTE]

>   > If you don't create the `sap-btp-manager` Secret, the BtpOperator CR is in the `Warning` state and the message is `Secret resource not found reason: MissingSecret`.

Reviewer: will you remember to update the message text here in case the message is changed? If possible, I'd try conveying the same content in a slightly vaguer way:

```suggestion
  > If you don't create the `sap-btp-manager` Secret, the BtpOperator CR warns you that the Secret is missing.
```
Out of curiosity - does it only warn you but keep functioning, or will it not work without the Secret?
